src/controller/AutoCommissioner.cpp

```diff
@@ -297,11 +297,14 @@ Optional<System::Clock::Timeout> AutoCommissioner::GetCommandTimeout(Commissioni
     // Per spec, all commands that are sent with the arm failsafe held need at least a 30s timeout.
     // Network clusters can indicate the time required to connect, so if we are connecting, use that time as long as it is > 30s.
     app::Clusters::NetworkCommissioning::Attributes::ConnectMaxTimeSeconds::TypeInfo::DecodableType seconds = 30;
+    // Chip-tool will wait for some time to receive connect network response.
+    app::Clusters::NetworkCommissioning::Attributes::ConnectMaxTimeSeconds::TypeInfo::DecodableType extraSeconds = 1;
+
     switch (stage)
     {
     case CommissioningStage::kWiFiNetworkEnable:
         ChipLogError(Controller, "Setting wifi connection time min = %u", mDeviceCommissioningInfo.network.wifi.minConnectionTime);
-        seconds = std::max(mDeviceCommissioningInfo.network.wifi.minConnectionTime, seconds);
+        seconds = std::max(static_cast<uint8_t>(mDeviceCommissioningInfo.network.wifi.minConnectionTime + extraSeconds), seconds);
```

Comment:
Yes, this part I don't get.  If a device claims it can take N seconds to connect to a network and get back to us, why are we adding padding to that?

Reply:
I think the device should be increasing the delay...